Rocket.Chat/API/API.swift

```diff
@@ -40,15 +40,15 @@ class API: APIFetcher {
     var userId: String?
 
     convenience init?(host: String, version: Version = .zero) {
-        guard let url = URL(string: host) else {
+        guard let url = URL(string: host)?.httpServerURL() else {
```

Comment:
We shouldn't make these changes here.

It would be better if we make this change where we are initializing the API object.

Take a look at APIExtensions.swift

Reply:
Done.